azure-pipelines/templates/package.yml

```diff
@@ -1,13 +1,13 @@
 steps:
```

Comment:
So is this file being used anywhere anymore?

Reply:
It's still being used by the old pipelines which I'd like to keep alive and working while we work on this.